vcbuild.bat

```diff
@@ -126,6 +126,8 @@ if "%target%"=="Clean" rmdir /S /Q %~dp0deps\icu
 
 call :getnodeversion || exit /b 1
 
+if "%target%"=="Clean" rmdir /Q /S "%~dp0%config%\node-v%FULLVERSION%-win-%target_arch%" > nul 2> nul
```

Comment:
On my box this deletes only `Release\node-v7.0.0-win-x64`, leaving other configurations